opensplice_cmake_module/cmake/Modules/FindOpenSplice.cmake

```diff
@@ -29,9 +29,14 @@ set(OpenSplice_FOUND FALSE)
 
 # check if provided OSPL_HOME is from an "official" binary package
 set(_ospl_home "$ENV{OSPL_HOME}")
-if(NOT "${_ospl_home} " STREQUAL " " AND
-    NOT EXISTS "${_ospl_home}/release.com")
-  set(_ospl_home "")
+if(WIN32)
+  if(NOT "${_ospl_home} " STREQUAL " " AND NOT EXISTS "${_ospl_home}/release.bat")
+    set(_ospl_home "")
+  endif()
+else()
+  if(NOT "${_ospl_home} " STREQUAL " " AND NOT EXISTS "${_ospl_home}/release.com")
+    set(_ospl_home "")
+  endif()
```

Comment:
Can you please reduce the duplicated logic by:

* set a variable to contain `release.bat` or `release.com` depending on the system

* have the logic only once using the variable